babel.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     [
       require.resolve('@babel/preset-env'),
       {
-        targets: {node: '18'},
+        targets: {node: '20'},
```

Comment:
Will RN 0.74 drop node 18? 0.73 targets node 18 as minimum so we need to keep this as 18, otherwise it’s a breaking change

Reply:
No, please look at the PR description.

Reply:
If the build step targets Node 20, running on Node 18 would be unsafe (even though it might work), so this *looks* like a breaking change to me?

Reply:
Ah, make sense 👍

Reply:
Ah, looks like `babel.config.js` only used for local development, and `tsc` (targeting ES2017) is used for npm builds? If that's the case this particular change isn't user facing.

*But* the bump of `@types/node` would allow the use of runtime features that aren't supported by Node 18. What's the motivation?

Reply:
The initial goal of this PR was to solve problem with CI failures on Node v20, but we solved by upgrading some dependencies, so this bump isn't necessary atm.